mautic/mautic
# minor fix for errors in recording UTM tags

UTM tag values of content, medium, source and term are overwriting UTM Campaign.
This change fixes that bug.

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | yes/no
| Automated tests included?              | no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     |  see description

#### Description:
UTM tag values of content, medium, source and term are overwriting UTM Campaign.
This change fixes that bug.

Add UTM parameters to a form action

<div id="mauticform_wrapper_utm" class="mauticform_wrapper">
   <form autocomplete="false" role="form" method="post" 
      action="https://mautic.local/form/submit?formId=1&utm_source=s1&utm_medium=m1">
           id="mauticform_utm" data-mautic-form="utm" enctype="multipart/form-data">
   .....
<form>
</div>


Fix errors in recording UTM tags
#### Steps to test this PR:
1. Create a form in mautic with email field
2. Embed its html in a html page served from another webserver (not mautic)
3. Add at least one UTM parameter, in addition to utm_campaign to the form action URL as shown above
4. Load the page in a browser and submit form
5. Look for UTM tags recorded event for the submitted contact
6. Notice that only UTM campaign tag is recorded and it is not the value of utm_campaign
